apps/browser-proxy/package.json

```diff
@@ -9,8 +9,10 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.645.0",
     "debug": "^4.3.7",
+    "expiry-map": "^2.0.0",
     "findhit-proxywrap": "^0.3.13",
-    "pg-gateway": "0.3.0-alpha.7",
+    "p-memoize": "^7.1.1",
+    "pg-gateway": "^0.3.0-alpha.7",
```

Comment:
Shouldn't this be 
```suggestion
    "pg-gateway": "^0.3.0-beta.2",
```

I believe alpha is pre-web APIs.

Reply:
I was working against my own branch of pg-gateway with the errors features and npm link. So I'll be that once my pg-gateway branch is merged and released.